README.md

```diff
@@ -47,6 +50,12 @@ yml file used to generate a code must have the same name as project. The file is
 
 ## Development
 
+###Jetty
+http://localhost:8080/kb-api-ws/swagger-ui/index.html <br>
```

Comment:
Man kan desværre ikke bruge "try it out"-funktionaliteten, når der er startet med `jetty:run-war`. Det er lidt ærgerligt, men ikke en blocker.

Reply:
Nej, hov, det virker jo helt som det skal! Jeg skulle bare vælge den rigtige indgang under `Servers`.